### PR TITLE
[docker] use a base image that already contains maven and openjdk

### PR DESCRIPTION
**Pull request by das7pad**
This PR makes use of the official maven docker image. This follows the docker principle of sharing common layers.

### CODE REVIEW

**Comment by JuneKelly** [comment]
Hi there, this change looks good to me :+1: , and thanks for contributing!

Before we can merge this in, we'll need you to sign our Contributor Licence Agreement, as described here: https://github.com/sharelatex/sharelatex/blob/master/CONTRIBUTING.md#contributor-license-agreement

Once that is done, just reply here and I'll pick up the next steps. 

Thanks again!

**Comment by das7pad** [comment]
I signed it yesterday.

**Comment by jdleesmiller** [comment]
👍 Thanks! Confirmed.

**Comment by JuneKelly** [comment]
Thanks @das7pad , I've merged this to an internal branch for a small adjustment and some more testing.

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,5 @@
 # Dockerfile for git-bridge
 
+FROM maven:3-jdk-8
```
Comment (JuneKelly) [inline Dockerfile:4]: I found that the container would crash pretty early with a `make not found` error. 

Adding this line:

```
RUN apt-get update && apt-get install -y git make curl
```
... solved the issue. I'll follow up on this on an internal branch before merging to master.

Comment (das7pad) [reply Dockerfile:4]: What are you using make for?

**Pull request merged**